qt/scientific_interfaces/Indirect/ConvFitDataTablePresenter.cpp

```diff
@@ -17,7 +17,7 @@ QStringList convFitHeaders() {
           << "WS Index"
           << "StartX"
           << "EndX"
-          << "Mask Energies";
+          << "Mask Q Range";
```

Comment:
My apologies I think my explanations have been unclear. Would it be better for this to be `Mask X Range` too (and for JumpFit) so that it is consistent with the `Mask X Range Of Spectrum` under the mini plots?

Reply:
I think keeping all of them the same makes sense